features/fixtures/mazerunner/jvm-scenarios/src/main/java/com/bugsnag/android/mazerunner/scenarios/HandledJavaSmokeScenario.java

```diff
@@ -24,11 +24,13 @@ public HandledJavaSmokeScenario(@NonNull Configuration config,
                                     @Nullable String eventMetadata) {
         super(config, context, eventMetadata);
         config.setAutoTrackSessions(false);
+
```

Comment:
This extra line seems to be failing the linter

Reply:
Fixed.